### PR TITLE
Fix GCR reference

### PR DESCRIPTION
**Pull request by tmatsuo**

### CODE REVIEW

**Comment by tmatsuo** [comment]
@kyessenov Unfortunately, the tests on travis don't actually test the change (the tests are skipped).

Also we don't run those tests on our jenkins either. @bshaffer Can you add configurations to our jenkins or travis to run those tests?

**Comment by bshaffer** [comment]
I'll run the tests locally for now. Looks like the Deployment Tests need to be updated for the latest endpoints version anyway.

File: endpoints/getting-started/test/DeployTest.php [hunk 1]
```diff
@@ -114,4 +107,22 @@ public function testEcho()
...
+        $response = $this->client->request(
+            'POST',
+            '/echo',
+            ['exceptions' => false]
+        );
+
+        $this->assertEquals(401, $response->getStatusCode());
+        $json = json_decode((string) $response->getBody(), true);
+        $this->assertArrayHasKey('message', $json);
+        $expectedString = 'Method doesn\'t allow unregistered callers ' .
```
Comment (tmatsuo) [inline endpoints/getting-started/test/DeployTest.php]: It's not resilient for string changes. How about just have a regex match with '/unregistered callers/i' or something like that?

**Comment by bshaffer** [comment]
I was able to deploy `container-engine.yaml` without any issues except for some usability ones I've filed bugs for! This can be merged once the tests pass.

**Comment by tmatsuo** [comment]
@bshaffer 

The test looks good overall. Can you loosen the assert a bit? https://github.com/GoogleCloudPlatform/php-docs-samples/pull/283#discussion_r93542060

File: endpoints/getting-started/test/DeployTest.php (hunk 1 above)
Comment (bshaffer) [inline endpoints/getting-started/test/DeployTest.php]: done

**Comment by tmatsuo** [comment]
LGTM

**Pull request merged**